From a05a63785c4769a1f3c3cddcc876c5acb6cf4c83 Mon Sep 17 00:00:00 2001
From: Justin Ruggles <justin.ruggles@gmail.com>
Date: Sat, 7 Apr 2012 12:55:12 -0400
Subject: [PATCH 221/278] daud: set channel layout

Update FATE reference due to WAVE channel mask now being set correctly.
---
 libavformat/daud.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/libavformat/daud.c b/libavformat/daud.c
index dda83f9..3ceb958 100644
--- a/libavformat/daud.c
+++ b/libavformat/daud.c
@@ -18,6 +18,8 @@
  * License along with Libav; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
+
+#include "libavutil/channel_layout.h"
 #include "avformat.h"
 
 static int daud_header(AVFormatContext *s) {
@@ -28,6 +30,7 @@ static int daud_header(AVFormatContext *s) {
     st->codec->codec_id = AV_CODEC_ID_PCM_S24DAUD;
     st->codec->codec_tag = MKTAG('d', 'a', 'u', 'd');
     st->codec->channels = 6;
+    st->codec->channel_layout = AV_CH_LAYOUT_5POINT1;
     st->codec->sample_rate = 96000;
     st->codec->bit_rate = 3 * 6 * 96000 * 8;
     st->codec->block_align = 3 * 6;
-- 
1.7.9.5

